cluster-agent/controllers/managed-gitops/eventloop/operation_event_loop_test.go

```diff
@@ -99,14 +102,20 @@ var _ = Describe("Operation Controller", func() {
 				},
 			}
 
+			namespaceToCreate := &corev1.Namespace{
```

Comment:
It looks like we're creating an Argo CD namespace here. If you're going to do that, you can remove the existing argocdNamespace field for this
```go
var argocdNamespace *corev1.Namespace
``` 
in this file.